sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/CBDE/CbdeHandler.cs

```diff
@@ -35,6 +35,44 @@
 
 namespace SonarAnalyzer.Rules.CSharp
 {
+    public class ThreadCpuStopWatch
+    {
+        private double totalMsStart;
+        private readonly ProcessThread currentProcessThread;
+        public long ElapsedMilliseconds
+        {
+            get { return (long)(currentProcessThread.TotalProcessorTime.TotalMilliseconds - totalMsStart); }
```

Comment:
you can replace this with arrow notation
```
public long ElapsedMilliseconds =>
	(long)(currentProcessThread.TotalProcessorTime.TotalMilliseconds - totalMsStart);
```

Reply:
done